parted: change check for device-manager

Other ptests use this method.

Upstream-Status: Submitted [bug-parted@gnu.org]

Signed-off-by: Joe Slater <joe.slater@windriver.com>


--- a/tests/t6001-psep.sh
+++ b/tests/t6001-psep.sh
@@ -19,7 +19,9 @@
 . "${srcdir=.}/init.sh"; path_prepend_ ../parted
 
 require_root_
-(dmsetup --help) > /dev/null 2>&1 || skip_test_ "No dmsetup installed"
+
+test "x$ENABLE_DEVICE_MAPPER" = xyes \
+  || skip_ "no device-mapper support"
 
 # Device maps names - should be random to not conflict with existing ones on
 # the system
